### PR TITLE
perf(fetch): add content body, when throw Error

### PR DESCRIPTION
**Pull request by ya-woodcutter**
1. Throw content with Error, check result.body on null or undefined
2. Remove default types in createAPI

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -808,6 +808,14 @@ babel-polyfill@^6.22.0:
     core-js "^2.4.0"
     regenerator-runtime "^0.10.0"
```
Comment (evilj0e) [inline yarn.lock]: Откуда он сам появился, если у тебя не изменились зависимости?

Comment (ya-woodcutter) [inline yarn.lock]: В предыдущем коммите Виталя апнул версию `babel-pollyfil` до 6.23.0.

File: src/createAPI.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ import applyMiddleware from './applyMiddleware';
 const createAPI = (
     resources = {},
     middleware = [],
-    APINamespace = '',
-    fetchOptions = {}
+    APINamespace,
+    fetchOptions
```
Comment (TimeRaider) [inline src/createAPI.js]: @Blackheart340 because of this, this should have been a major version bump

Comment (ya-woodcutter) [inline src/createAPI.js]: @TimeRaider generally it's a bug, because the documentation says completely different default values :)

Comment (TimeRaider) [inline src/createAPI.js]: true, but doc changes do not break consumer's code, code changes do

Comment (Blackheart340) [inline src/createAPI.js]: @TimeRaider fine, so I can unpublish latest version and bump major

Comment (TimeRaider) [inline src/createAPI.js]: @Blackheart340 what was the reasoning for removing defaults in a first place?

Comment (Blackheart340) [inline src/createAPI.js]: @TimeRaider 'cause it's overriding default values in `callAPIMethod` and it's not expected behavior

Comment (TimeRaider) [inline src/createAPI.js]: @Blackheart340 so maybe the order was wrong?

Comment (Blackheart340) [inline src/createAPI.js]: @TimeRaider what do you mean?
We have some default options in `callApiMethod` and you defined the same options higher without any reason. It isn't necessary default options there and now we have wrong behavior. Aren't we?

Comment (TimeRaider) [inline src/createAPI.js]: @Blackheart340 provide a failing test

**Pull request merged**